File: web_benchmark/test/get/Dockerfile.java
FROM eclipse-temurin:17-jdk-alpine

WORKDIR /app

# ก๊อปปี้ไฟล์โค้ดเข้ามา
COPY server.java .

# คอมไพล์โค้ด
RUN javac server.java

# เปิดพอร์ต
EXPOSE 8080

# สั่งรัน
CMD ["java", "server"]

File: main_web_benchmark/GET/get_with_index/Dockerfile.go
FROM golang:1.21-alpine AS builder
WORKDIR /app
COPY go.mod go.sum* ./
RUN go mod download
COPY server.go .
RUN CGO_ENABLED=0 GOOS=linux go build -o server server.go

FROM alpine:latest
WORKDIR /app
COPY --from=builder /app/server .
EXPOSE 8004
CMD ["./server"]
